packages/adp-tooling/src/types.ts

```diff
@@ -39,3 +39,45 @@ export interface AdpWriterConfig {
         description?: string;
     };
 }
+
+export interface ManifestAppdescr {
+    fileName: string;
+    layer: string;
+    fileType: string;
+    reference: string;
+    id: string;
+    namespace: string;
+    version: string;
+    content: Content[];
+}
+
+export interface Content {
+    changeType: string;
+    content: object;
+    texts?: object;
+}
+
+export const enum FolderNames {
+    Webapp = 'webapp',
```

Comment:
`Webapp` is never used in your code (and also shouldn't be used), therefore, please remove it.